helpers/addresses.py

```diff
@@ -716,7 +720,13 @@
         "ops_deployer": "0xDA25ee226E534d868f0Dd8a459536b03fEE9079b",
         "ops_deployer2": "0xeE8b29AA52dD5fF2559da2C50b1887ADee257556",
         "ops_deployer6": "0x96AC69183216074dd8CFA7A380e873380445EaDc",
+        "ops_executor1": "0xcf4fF1e03830D692F52EB094c52A5A6A2181Ab3F",
+        "ops_executor2": "0x8938bf50d1a3736bdA413510688834540858dAEA",
+        "ops_executor3": "0xC69Fb085481bC8C4bfF99B924076656305D9a25D",
+        "ops_executor4": "0xBB2281cA5B4d07263112604D1F182AD0Ab26a252",
         "ops_executor6": "0x66496eBB9d848C6A8F19612a6Dd10E09954532D0",
```

Comment:
nit but not sure if you want to keep this clean, you go from executor 4 to 6 after the delete.

Reply:
this is to stay consistent with mainnet/other chains's labels.